worker/codalabworker/aws_batch.py

```diff
@@ -67,6 +71,7 @@ def gpus(self):
         return 0
 
     def create_run(self, bundle, bundle_path, resources):
+        resources['request_memory'] = resources.get('request_memory') or BATCH_DEFAULT_MEMORY
```

Comment:
should this be `resources['request_memory'] = resources.get('request_memory', BATCH_DEFAULT_MEMORY)` or is it like 0 or something?

Reply:
It is always set but set to None :(